src/tests/Microsoft.PowerFx.Interpreter.Tests/RecalcEngineTests.cs

```diff
@@ -749,22 +749,22 @@ public void CheckLambdaBindError()
         public void CheckDottedBindError()

```

Comment:
>CheckDottedBindError [](http://example.com/codeflow?start=20&length=20)

This is the same test as the one in BindingEngineTests.cs in core. Do we need both?

Reply:
Its because engines are different, engine vs recalcengine.